packages/sitecore-jss/package.json

```diff
@@ -52,13 +52,15 @@
     "typescript": "~4.3.5"
   },
   "dependencies": {
+    "@types/sinon": "^17.0.3",
     "axios": "^0.21.1",
     "chalk": "^4.1.0",
     "debug": "^4.3.1",
     "graphql": "^16.5.0",
     "graphql-request": "^4.2.0",
     "lodash.unescape": "^4.0.1",
     "memory-cache": "^0.2.0",
+    "sinon": "^17.0.1",
```

Comment:
It's better to have this as a dev dependency - or use an already imported module alternative, if possible